### PR TITLE
(SIMP-2537) Changelog update for 6.0.0-Beta

### PR DESCRIPTION
**Pull request by trevor-vaughan**
SIMP-2537 #close

### CODE REVIEW

File: Changelog.rst [hunk 1]
```diff
@@ -59,8 +65,22 @@ auto-update for all future RPM updates. If you wish to disable this behavior,
 you should edit the options in ``/etc/simp/adapter_config.yaml``.
 
 .. NOTE::
-   Anything that is in a Git or Subversion repository in the `simp` environment
-   will **not** be overwritten by ``simp_rpm_helper``.
+   Anything that is in a Git or Subversion repository in the ``simp`` environment
```
Comment (jeefberkey) [inline Changelog.rst]: [SIMP-2261](https://simp-project.atlassian.net/browse/SIMP-2261) should block this.

File: Changelog.rst [hunk 2]
```diff
@@ -85,15 +122,33 @@ A simple find and replace of ``concat_fragment`` and ``concat_build`` in legacy
...
+The ``openldap`` module has been renamed to ``simp_openldap`` to pave the way
+towards using a more up-to-date implementation of the core openldap component
+module from the community.
+
+pupmod-simp-snmpd
+"""""""""""""""""
+
+The ``snmpd`` module has been removed until updates can be made available
 
 Puppet AIO
```
Comment (jeefberkey) [inline Changelog.rst]: [SIMP-2263](https://simp-project.atlassian.net/browse/SIMP-2263) covers this.

File: Changelog.rst [hunk 3]
```diff
@@ -112,6 +112,38 @@ For instance, ``iptables::tcp_stateful_listen`` is now ``iptables::listen::tcp_s
 Additionally, any ``add_rule`` defines were changed to just ``rule``. For
 example, ``auditd::add_rule`` was changed to just ``auditd::rule``.
 
+Centralized Management of Application x509 PKI Certs
```
Comment (jeefberkey) [inline Changelog.rst]: 👍 Added a ticket to add more like this to the real docs: https://simp-project.atlassian.net/browse/SIMP-2544

Comment (nick-markowski) [inline Changelog.rst]: Nice! I meant to do that...thanks.

File: Changelog.rst [hunk 4]
```diff
@@ -85,15 +283,49 @@ A simple find and replace of ``concat_fragment`` and ``concat_build`` in legacy
...
+
+Module has been rewritten and renamed to pupmod-simp-gdm.
+
+ELG Stack
+"""""""""
 
-The Foreman module has been removed until it works consistently with Puppet 4
+pupmod-simp-logstash now uses TLS.
 
 Puppet AIO
```
Comment (nick-markowski) [inline Changelog.rst]: This is also referenced at the top, and should probably be moved up there for consistency's sake

File: Changelog.rst (hunk 1 above)
Comment (op-ct) [inline Changelog.rst]: Maybe the RC

**Pull request merged**